### ISSUE TITLE
Make copy + paste into REPL easier by giving option to autowrite function name

### ISSUE THREAD

**Issue by anandijain** [elm/compiler#2030]
When defining a function in the REPL, the REPL will auto write the function name for you, this is actually more of an impedance for me than it is useful.

Often I want to test something by copy-pasting functions in, but I can't because I have to write the type annotation then copy the rest so that I don't get a `Bad Type Annotation` REPL error.

Anyone else wish they had the option to disable this repl feature?

**Comment by rlefevre** [comment]
@anandijain Can't you copy the function without the type annotation? 
This should work and the type will be inferred anyway.

**Comment by anandijain** [comment]
Yes, thank you, but for large blocks of code ~30-50 lines with multiple function definitions.

**Comment by edwardabraham** [comment]
Yep. I am just learning Elm, and want to paste blocks of code into the REPL to test them and to try to figure things out. This gets in the way.
